pipeline_steps/deploy.groovy

```diff
@@ -55,11 +55,22 @@ def upgrade(String stage_name, String upgrade_script, List env_vars) {
           sh "git reset --hard"
         }
         common.prepareRpcGit()
```

Comment:
@weezer I think the problem @git-harry is pointing out is that common.prepareRpcGit will prepare /opt/rpc-openstack on the host. If that is not copied into deploy1 vm, then there is no point in running the upgrade script on deploy1 as it will still have the same version of rpc-openstack that was deployed the first time.